### PR TITLE
cmd/protoc-gen-go-grpc: add protoc and protoc-gen-go-grpc versions to top comment

### LINKED ISSUE #4303: Add protoc-gen-go-grpc and protoc versions
Hi there,

It would be helpful to have protoc-gen-go-grpc and protoc versions in the top comment in files generated protoc-gen-go-grpc because it helps to track version differences in change requests.

Just like what protoc-gen-go already [does](https://github.com/protocolbuffers/protobuf-go/blob/31610fd9139106e5b7432e3533ca1254bfa32433/cmd/protoc-gen-go/internal_gengo/main.go#L125)

If this accepted I can implement it.

### PR DESCRIPTION
**Pull request by amenzhinsky**
Closes #4303

### CODE REVIEW

**Comment by amenzhinsky** [comment]
Not sure whether I need regenerate proto files.

**Comment by menghanl** [comment]
> Not sure whether I need regenerate proto files.

Yes. Please run https://github.com/grpc/grpc-go/blob/master/regenerate.sh.
This should also fix the failures in `Testing / vet-proto`

**Comment by amenzhinsky** [comment]
@menghanl done

**Review by menghanl — APPROVED** [review]
This looks good to me. Thanks for the change.

Assigning to @dfawley to also take a look.

File: cmd/protoc-gen-go-grpc/grpc.go [hunk 1]
```diff
@@ -43,13 +42,29 @@ func generateFile(gen *protogen.Plugin, file *protogen.File) *protogen.Generated
 	filename := file.GeneratedFilenamePrefix + "_grpc.pb.go"
 	g := gen.NewGeneratedFile(filename, file.GoImportPath)
 	g.P("// Code generated by protoc-gen-go-grpc. DO NOT EDIT.")
+	g.P("// versions:")
+	g.P("// \tprotoc-gen-go-grpc v", version)
+	g.P("// \tprotoc             v", protocVersion(gen))
```
Comment (dfawley) [inline cmd/protoc-gen-go-grpc/grpc.go:47]: Let's replace the `\t` with `- ` instead.

Comment (amenzhinsky) [reply cmd/protoc-gen-go-grpc/grpc.go:47]: I simply copied what protoc-gen-go does to match its header, are you sure about this?

Comment (dfawley) [reply cmd/protoc-gen-go-grpc/grpc.go:47]: Yes, I noticed that.  Please change it, thank you.

**Review by dfawley — APPROVED** [review]

**Comment by dfawley** [comment]
Thank you for the PR, @amenzhinsky!

**Pull request merged**